views/partials/applicationModal.hbs

```diff
@@ -2,17 +2,10 @@
     <div class="modal-content container-fluid">
```

Comment:
You don't need `.container-fluid` here. It just add white space around the sides, which we don't want.